x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/routes/indices.ts

```diff
@@ -102,12 +103,14 @@ async function getIndicesFromPattern(
       },
     },
   };
-  const response = await dataClient.callAsCurrentUser('search', params);
-  if (response.status === 404 || !response.aggregations) {
+  const response: SearchResponse<unknown> = await dataClient.callAsCurrentUser('search', params);
```

Comment:
ya, this client gets hard to reason about, as there are options to have it return different things on 404's, and what not - it's not even all just the client, ES has such options you can send on requests.  Leaving the TODO comment is good for next time we need to look into this particular site ...

Reply:
👍